config/crds/vsphere_v1alpha1_vspheremachineproviderspec.yaml

```diff
@@ -342,7 +330,7 @@ spec:
           properties:
             devices:
               description: Devices is the list of network devices used by the virtual
-                machine.
+                machine. TODO(akutz) Make sure at least one network matches the             ClusterSpec.CloudProviderConfiguration.Network.Name
```

Comment:
remove the extra space?

Reply:
That can be addressed in a future PR/patch. Thanks!